imap_processing/cdf/utils.py

```diff
@@ -41,6 +42,24 @@ def calc_start_time(shcoarse_time: float) -> np.datetime64:
     return launch_time + time_delta
 
 
+def load_cdf(file_path: Path, **kwargs) -> xr.Dataset:
+    """Load the contents of a CDF file into an ``xarray`` dataset.
+
+    Parameters
+    ----------
+    file_path : Path
+        The path to the CDF file
+    kwargs : dict
+        Keyword arguments for ``cdf_to_xarray``
+
+    Returns
+    -------
+    dataset : xr.Dataset
```

Comment:
Since this doesn't have a named variable I think it should be dropped? But I don't know for sure.
https://numpydoc.readthedocs.io/en/latest/format.html#returns
```suggestion
    xr.Dataset
```
Also update the `write_cdf()` returns section if you take this.

Reply:
Yes, good catch.